js/lib/config.js

```diff
@@ -1,3 +1,6 @@
+/**
+ * This file is a copy of shared/js/contacts/import/utilities/config.js in gaia.
+ */
```

Comment:
If this is really a copy, then leave this file alone so it remains a copy, and put that comment in the README please.